.pre-commit-config.yaml

```diff
@@ -63,6 +63,9 @@ repos:
     rev: v2.15.5
     hooks:
     -   id: pylint
+    -   language: system
+    -   pass_filenames: false
+    -   stages: [manual]
```

Comment:
this isn't quite right, check the mypy hook for an example of the syntax

Reply:
I'm sorry, are the changes ok now?

Reply:
should be fine, thanks!